.github/workflows/dab-release.yml

```diff
@@ -0,0 +1,47 @@
+---
+name: DAB requirements update
```

Comment:
```suggestion
name: django-ansible-base requirements update
```

Reply:
I chose DAB here because github actions workflow name space is limited.

It would probably look like:

`django-ansible-base requirements up...`

![image](https://github.com/user-attachments/assets/e3aa2dc5-9cda-4cc7-8210-aad6ef6762de)